system-contract-dapp-playground/src/components/contract-interaction/erc/erc-20/methods/BalanceOf.tsx

```diff
@@ -0,0 +1,109 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { useState } from 'react';
+import { Tooltip } from '@chakra-ui/react';
+import { AiOutlineMinus } from 'react-icons/ai';
+import { ContractFactory, BaseContract } from 'ethers';
+import HederaCommonTextField from '@/components/common/HederaCommonTextField';
+
+interface PageProps {
+  contractFactory: ContractFactory<any[], BaseContract>;
+}
+
+const BalanceOf = ({ contractFactory }: PageProps) => {
+  const [value, setValue] = useState('');
+  const [balances, setBalances] = useState([
```

Comment:
Remove these defaults. It's enough to have shown it in the video, no need to check it in

Reply:
Yes, these will be updated to retrieve dynamic values from the onchain smart contracts

Reply:
Okay, will wait on new PR